test.js

```diff
@@ -18,3 +18,51 @@ test(function (t) {
 		t.assert(code === 0);
 	});
 });
+
+test('--json', function (t) {
+	t.plan(1);
+
+	var cp = childProcess.spawn('./cli.js', ['--json'], {
+		cwd: __dirname
+	});
+
```

Comment:
Add `cp.stdout.setEncoding('utf8')` here and you don't have to toString it later.